src/common/interfaces.ts

```diff
@@ -20,7 +20,13 @@ export interface UnprocessableEntityError {
 
 // Channel types supported in Knock
 // TODO: it would be great to pull this in from an external location
```

Comment:
could we get these from providers? https://github.com/knocklabs/providers-ex/blob/main/lib/providers.ex

i assume this isn't in scope of this PR so not blocking

Reply:
Good question, I've never tried something like that, but I agree it'd be good to have a single place where we keep our list of providers.